src/Queries/GetApprenticeshipsByAcademicYear/GetApprenticeshipsByAcademicYearQueryHandler.cs

```diff
@@ -1,23 +1,26 @@
+using SFA.DAS.Apprenticeships.Domain;
 using SFA.DAS.Apprenticeships.Domain.Repositories;
 
-namespace SFA.DAS.Apprenticeships.Queries.GetApprenticeshipsByDates;
+namespace SFA.DAS.Apprenticeships.Queries.GetApprenticeshipsByAcademicYear;
 
 public class GetApprenticeshipsByAcademicYearQueryHandler(
     IApprenticeshipQueryRepository queryRepository)
-    : IQueryHandler<GetApprenticeshipsByDatesRequest, GetApprenticeshipsByDatesResponse>
+    : IQueryHandler<GetApprenticeshipsByAcademicYearRequest, GetApprenticeshipsByAcademicYearResponse>
 {
-    public async Task<GetApprenticeshipsByDatesResponse> Handle(GetApprenticeshipsByDatesRequest query, CancellationToken cancellationToken = default)
+    public async Task<GetApprenticeshipsByAcademicYearResponse> Handle(GetApprenticeshipsByAcademicYearRequest query, CancellationToken cancellationToken = default)
     {
+        var academicYearDates = AcademicYearParser.ParseFrom(query.AcademicYear);
+        
         var response  = await queryRepository.GetByDates(
             query.UkPrn,
-            query.Dates, 
+            academicYearDates, 
             query.Page, 
```

Comment:
Small observation that may not relate directly to this PR. The Page parameter does not appear to be used by the `GetByDates` method, other than to return it back again to form part of the response, since the `limit` to be applied has already been calculated. This parameter could therefore be removed for clarity, and the request value fed directly into the response.